<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistEditDialog</class>
 <widget class="QDialog" name="PlaylistEditDialog">
  <property name="windowTitle">
   <string>Плейлист</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QPushButton" name="btnSelectCover">
     <property name="text">
      <string>Выбрать обложку</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="labelCoverPreview">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLineEdit" name="lineEditTitle">
     <property name="placeholderText">
      <string>Название плейлиста</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QTextEdit" name="textEditDescription">
     <property name="placeholderText">
      <string>Описание...</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
